samples/bluetooth/hci_uart/debug.conf

```diff
@@ -0,0 +1,22 @@
+CONFIG_ASSERT=y
```

Comment:
Could you please update the sample's `README.rst` with info on how to use this `debug.conf`?